src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -16,7 +16,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9"
+        "php": ">=5.3.3 EVEN ON LATEST SYMFONY VERSIONS TO ALLOW USING",
+        "php": "THIS BRIDGE WHEN TESTING LOWEST SYMFONY VERSIONS.",
```

Comment:
@xabbuh your proposal would not work here, as we are inside ``require``, which is a map, not an object where composer will only read keys it knows.
thus, even using ``_`` at the top level would be a bad idea: the composer.json would not validate against the schema anymore (composer uses it in the lock file, which is not validated against a schema).